src/environments/.env.ts

```diff
@@ -2,8 +2,8 @@
 /* tslint:disable */
 export default {
   'mifos_x': {
-    'version': '0.0.0',
```

Comment:
do we need these changes?

Reply:
@adamsaghy I didn't make these changes, looks like auto generated.